components/dropdown/dropdown-menu.js

```diff
@@ -12,14 +15,58 @@ import { ThemeMixin } from '../../mixins/theme/theme-mixin.js';
  */
 class DropdownMenu extends ThemeMixin(DropdownContentMixin(LitElement)) {
 
+	static get properties() {
+		return {
+			/**
+			 * @ignore
+			 */
+			_closeRadio: {
+				type: Boolean,
+				reflect: true,
+				attribute: '_close-radio'
+			},
+		};
+	}
+
 	static get styles() {
-		return dropdownContentStyles;
+		return [
+			dropdownContentStyles,
+			css`
+				:host {
+					--d2l-dropdown-close-animation-name: d2l-dropdown-close-animation;
+				}
+
+				:host([theme="dark"]) {
+					--d2l-dropdown-close-animation-name: d2l-dropdown-close-animation-dark;
+				}
+
+				:host([_close-radio]) {
+					animation: var(--d2l-dropdown-close-animation-name) 300ms ease-out;
+				}
+
+				@media (prefers-reduced-motion: reduce) {
+					:host([_close-radio]) {
+						animation: none !important;
+					}
+				}
+				@keyframes d2l-dropdown-close-animation {
+					0% { display: inline-block; opacity: 1; transform: translate(0, 0); }
+					100% { display: none; opacity: 0; transform: translate(0, -10px); }
+				}
+
+				@keyframes d2l-dropdown-close-animation-dark {
+					0% { display: inline-block; opacity: 0.9; transform: translate(0, 0); }
+					100% { display: none; opacity: 0; transform: translate(0, -10px); }
```

Comment:
I'm wondering if it's possible to change this such that we don't need to modify `display` within these animations?